script/script.js

```diff
@@ -143,10 +151,37 @@ function buildDetails(index) {
   for (let i = 0; i < arrChild.length; i += 1) {
     contenainer.appendChild(arrChild[i]);
   }
+  articleContainer.appendChild(contenainer);
+  divCotainer.appendChild(articleContainer);
+  sectionRecentWork.appendChild(divCotainer);
 }
 // put project section in a variable to add a listener
 const projectSection = document.querySelector('.box-articles');
 projectSection.addEventListener('click', (event) => {
   const index = event.target.id;
   buildDetails(index);
 });
+
+// FORM VALIDATION
+const email = document.getElementById('email');
+const form = document.getElementById('form-contact');
+const errorScreen = document.getElementById('errorMsg');
+
+function isEmailValid(email) {
+  const regex = /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/;
+  return regex.test(String(email));
+}
+
+function emailValidation() {
+  const userEmail = email.value.trim();
+  if (!isEmailValid(userEmail)) {
+    errorScreen.textContent = 'Please Email must be all in lowercase';
+    email.classList.add('error');
+    email.classList.remove('success');
+  }
+}
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  emailValidation();
+});
```

Comment:
- [ ] You did a great job working on this milestone! 🚀 However, there's a problem trying to submit a valid email. This happens because you're preventing the form's default behavior and running the `emailValidation` function which only handles the error case. I suggest you add an `else` clause that runs if the email **IS** valid and submits the form, granting the expected behavior of this feature! 🤓 🛠️ 

Something like this:

```suggestion
function emailValidation() {
  const userEmail = email.value.trim();
  if (!isEmailValid(userEmail)) {
    errorScreen.textContent = 'Please Email must be all in lowercase';
    email.classList.add('error');
    email.classList.remove('success');
  } else { // This runs if the email is valid
    errorScreen.textContent = '';
    email.classList.add('success');
    email.classList.remove('error');
    form.submit()
  }
}

form.addEventListener('submit', (event) => {
  event.preventDefault();
  emailValidation();
});

Please consider applying this! 🤓 
```

Reply:
@anagudelogu, Thank you very much... You're the best 😃